public/node-releases-data.json

```diff
@@ -1 +1 @@
e":"2022-06-01","modules":"102"},{"major":16,"version":"16.20.1","codename":"Gallium","currentStart":"2021-04-20","ltsStart":"2021-10-26","maintenanceStart":"2022-10-18","endOfLife":"2023-09-11","npm":"8.19.4","v8":"9.4.146.26","releaseDate":"2023-06-20","modules":"93"},{"major":15,"version":"15.14.0","currentStart":"2020-10-20","maintenanceStart":"2021-04-01","endOfLife":"2021-06-01","npm":"7.7.6","v8":"8.6.395.17","releaseDate":"2021-04-06","modules":"88"},{"major":14,"version":"14.21.3","codename":"Fermium","currentStart":"2020-04-21","ltsStart":"2020-10-27","maintenanceStart":"2021-10-19","endOfLife":"2023-04-30","npm":"6.14.18","v8":"8.4.371.23","releaseDate":"2023-02-16","modules":"83"},{"major":13,"version":"13.14.0","currentStart":"2019-10-22","maintenanceStart":"2020-04-01","endOfLife":"2020-06-01","npm":"6.14.4","v8":"7.9.317.25","releaseDate":"2020-04-29","modules":"79"},{"major":12,"version":"12.22.12","codename":"Erbium","currentStart":"2019-04-23","ltsStart":"2019-10-21","maintenanceStart":"2020-11-30","endOfLife":"2022-04-30","npm":"6.14.16","v8":"7.8.279.23","releaseDate":"2022-04-05","modules":"72"},{"major":11,"version":"11.15.0","currentStart":"2018-10-23","maintenanceStart":"2019-04-22","endOfLife":"2019-06-01","npm":"6.7.0","v8":"7.0.276.38","releaseDate":"2019-04-30","modules":"67"},{"major":10,"version":"10.24.1","codename":"Dubnium","currentStart":"2018-04-24","ltsStart":"2018-10-30","maintenanceStart":"2020-05-19","endOfLife":"2021-04-30","npm":"6.14.12","v8":"6.8.275.32","releaseDate":"2021-04-06","modules":"64"},{"major":9,"version":"9.11.2","currentStart":"2017-10-01","maintenanceStart":"2018-04-01","endOfLife":"2018-06-30","npm":"5.6.0","v8":"6.2.414.46","releaseDate":"2018-06-12","modules":"59"},{"major":8,"version":"8.17.0","codename":"Carbon","currentStart":"2017-05-30","ltsStart":"2017-10-31","maintenanceStart":"2019-01-01","endOfLife":"2019-12-31","npm":"6.13.4","v8":"6.2.414.78","releaseDate":"2019-12-17","modules":"57"},{"major":7,"version":"7.10.1","currentStart":"2016-10-25","maintenanceStart":"2017-04-30","endOfLife":"2017-06-30","npm":"4.2.0","v8":"5.5.372.43","releaseDate":"2017-07-11","modules":"51"},{"major":6,"version":"6.17.1","codename":"Boron","currentStart":"2016-04-26","ltsStart":"2016-10-18","maintenanceStart":"2018-04-30","endOfLife":"2019-04-30","npm":"3.10.10","v8":"5.1.281.111","releaseDate":"2019-04-03","modules":"48"},{"major":5,"version":"5.12.0","currentStart":"2015-10-29","maintenanceStart":"2016-04-30","endOfLife":"2016-06-30","npm":"3.8.6","v8":"4.6.85.32","releaseDate":"2016-06-23","modules":"47"},{"major":4,"version":"4.9.1","codename":"Argon","currentStart":"2015-09-08","ltsStart":"2015-10-12","maintenanceStart":"2017-04-01","endOfLife":"2018-04-30","npm":"2.15.11","v8":"4.5.103.53","releaseDate":"2018-03-29","modules":"46"},{"major":0,"version":"0.12.18","currentStart":"2015-02-06","endOfLife":"2016-12-31","npm":"2.15.11","v8":"3.28.71.20","releaseDate":"2017-02-22","modules":"14"}]
```

Comment:
Hey @ovflowd  I've been a bit busy lately. Is that normal?

Reply:
as far as I know, no, those should be reverted during precommit

in a different thread it was suggested to check your  precommit install. running `npm install` *should* re-install husky.

https://github.com/nodejs/nodejs.org/blob/main/.husky/pre-commit#L6-L8

perhaps if that doesn't work we should investigate why

Reply:
Yes not normal. For me, and pretty much everyone the hooks are working as expected. I'd recommend recloning the repository as last effort measure.